src/osx/cocoa/dataview.mm

```diff
@@ -1653,8 +1499,6 @@ -(id) init
         currentlyEditedColumn =
             currentlyEditedRow = -1;
 
-        [self setAutoresizesOutlineColumn:NO];
-        [self registerForDraggedTypes:[NSArray arrayWithObjects:DataViewPboardType,NSStringPboardType,nil]];
```

Comment:
I _guess_ this line was removed intentionally, so I'm not changing anything here, but please double check it to confirm.